omniscidb/ArrowStorage/ArrowStorage.cpp

```diff
@@ -470,14 +470,19 @@ void ArrowStorage::appendArrowTable(std::shared_ptr<arrow::Table> at, int table_
   for (auto& frag : fragments) {
     frag.metadata.resize(at->columns().size());
   }
-
```

Comment:
Please try to avoid unrelated changes. Do you intentionally remove this vertical space? If yes, then what for?

Reply:
ok, I will remove this change.